pages/add/form/detail-info-fields.js

```diff
@@ -22,6 +22,32 @@ let Tags = React.createClass({
       .catch((reason) => {
         console.error(reason);
       });
+
+      this.shimTagDelimiter();
+  },
+  shimTagDelimiter: function() {
+    // The ReactTags component for some reason does not
+    // do tag seperation on anything except "enter" which is
+    // not an intuitive key when you're on a single-line input.
+    try {
+      const reactTags = this.reactTags;
+      const trueInput = reactTags.input.input;
+      const forceKey = reactTags.handleKeyDown.bind(reactTags);
+
+      trueInput.addEventListener("keyup", evt => {
+        // did we type a comma?
+        const key = evt.key;
+        if (key === ',') {
+          // remove the comma and type an enter instead
+          const { query, selectedIndex } = reactTags.state;
+          reactTags.state.query = query.substring(0, query.length-1).trim();
```

Comment:
This isn't the right spot for this, doesn't get triggered on the final tag if the user separates with tab/return